documentation/src/main/asciidoc/user_guide/eviction.adoc

```diff
@@ -229,7 +258,7 @@ Eviction is disabled by default. Default values are used:
 * size: -1 is used if not specified, which means unlimited entries.
 * 0 means no entries, and the eviction thread will strive to keep the cache empty.
 
-Expiration lifespan and maxIdle both default to -1.
+Expiration lifespan and maxIdle both default to -1, which means that the entry is never removed due to expiration.
```

Comment:
"which means that entries will be created immortal by default. This can be overridden per entry with the API"